core/application/tests/userWorkflows/userMessages.test.ts

```diff
@@ -0,0 +1,35 @@
+import { getEmailVerificationMessage, getPasswordResetVerificationMessage } from '@application/userWorkflows/userMessages'
+import { GenericMessage } from '@commons/dto/MessageDTO'
+
+describe('User Message Functions', () => {
+  const userName = 'Ebrahim'
+  const securityCode = '123456'
+
+  test('should generate email verification message', () => {
+    const expectedMessage: GenericMessage = {
+      title: 'Welcome to Blood Connect!',
+      content: `Hello ${userName},<br/><br/>
+              Welcome! Please verify your email using the following code: ${securityCode}.<br/><br/>
+              Thanks!`
+    }
+
+    const result = getEmailVerificationMessage(userName, securityCode)
+
+    expect(result).toEqual(expectedMessage)
+  })
+
+  test('should generate password reset verification message', () => {
+    const expectedMessage: GenericMessage = {
+      title: 'Reset your password for Blood Connect',
+      content: `Hello ${userName},<br/><br/>
```

Comment:
move this in a const file

Reply:
Move the title and content to a separate file.
You will find with this [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/6d856e5322c15066df61036318febceea7de3529)